behave.ini.dist-cerner

```diff
@@ -3,8 +3,12 @@ logging_level=DEBUG
 logging_format=%(message)s
 
 [api]
-url=https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/
-patient=1696023
+url=https://fhir.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/
+patient=1316024
 
 [auth]
-strategy=none
+strategy=refresh_token
+refresh_token=69746025-cd6b-4129-ac7b-b11d04f625b5
```

Comment:
It doesn't really. I'm thinking that a better solution is actually a "launch/authorize" button on the testing site that walks you through the authorization process and saved a refresh token in a `.refresh_token` file or something like that. That would be another better fix than what we currently have on the road to the webdriver, and most of the logic (like what would power the `/authorized/` endpoint) would actually end up being useful to the webdriver.